src/approximations/nystrom.jl

```diff
@@ -0,0 +1,103 @@
+using StatsBase
+using LinearAlgebra
+# Following the algorithm by William and Seeger, 2001
+# Cs is equivalent to X_mm and C to X_mn
+
+function sampleindex(X::AbstractMatrix, r::AbstractFloat; obsdim::Integer=defaultobs)
```

Comment:
Maybe change this to
```suggestion
function sampleindex(X::AbstractMatrix, r::Real; obsdim::Integer=defaultobs)
```
?